cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -643,6 +643,57 @@ <h2>Fees</h2>
             {% endfor %}
         {% endif %}
     </dl>
+    <h2>Rewards and perks</h2>
+    <dl class="m-list m-list__card-details">
+        <dt>Rewards</dt>
+        <dd>
+            {% if card.rewards %}
+            <ul>
+                {% for reward in card.rewards %}
+                    <li>
+                        {{ 'Cash back' if reward == 'Cashback rewards' }}
+                        {{ 'Travel-related rewards' if reward == 'Travel-related rewards' }}
+                        {{ card.other_rewards if reward == 'Other rewards' }}
+                    </li>
+                {% endfor %}
+            </ul>
+            {% else %}
+                None
+            {% endif %}
+        </dd>
+        {% if card.services %}
+            <dt>Services</dt>
+            <dd>
+                <ul>
+                    {% for service in card.services %}
+                        <li>
+                            {% if service != 'Other' %}
+                                {{ service }}
+                            {% else %}
+                                {{ card.other_services }}
+                            {% endif %}
```

Comment:
This could be expressed more concisely if you wish, similar below for card features as well:

```suggestion
                            {{ service if service != 'Other' else card.other_services }}
```